openpype/hosts/blender/api/render_lib.py

```diff
@@ -232,11 +232,11 @@ def prepare_rendering(asset_group):
     set_render_format(ext, multilayer)
     aov_list, custom_passes = set_render_passes(settings)
 
-    output_path = os.path.join(file_path, render_folder, file_name)
+    output_path = Path.joinpath(file_path, render_folder, file_name)
```

Comment:
Likely personal preference but this should also work, right?
```suggestion
    output_path = file_path / render_folder / file_name
```

Reply:
Yes it's the same, I like `joinpath` even if longer because it's more clear :) but I can be convinced to change it :P